scripts/control_device.py

```
@@ -582,6 +583,19 @@
     help="the circuit number to turn on",
 )
 
+# get_heater_state parser
+_get_heater_state_examples = """example usage:
+
+poetry run control_device get_heater_state -c "heater" -k "zvVvd7JxtN7CgvkD1Psujw==" -d ab1c2d -i "111.222.11.22"\n
+"""  # noqa E501
```

Ruff (0.14.13)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)